src/test/factories/cycle.js

```diff
@@ -14,5 +17,6 @@ export default function define(factory) {
     state: CYCLE_STATES[0],
     createdAt: cb => cb(null, now),
     updatedAt: cb => cb(null, now),
+    weekStartedAt: cb => cb(null, startOfWeek),
```

Comment:
Since the `weekStartedAt` field is not officially in the cycle model (and is only a derived value provided by the graphql API for the Cycle schema), we don't really need this prop here. But if it stays, it should at the very least return the raw date so that anything else using this factory can use that value.